migrations/Version202201121636313291_ltiTestReview.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\ltiTestReview\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use common_ext_Extension as Extension;
+use common_ext_ExtensionsManager as ExtensionsManager;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202201121636313291_ltiTestReview extends AbstractMigration
+{
+    private const CONFIG_FILE = 'ReviewPanel';
+
+    public function getDescription(): string
+    {
+        return 'Set ReviewPanel config values';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // set ReviewPanel.conf values
+        $extension = $this->getExtension();
+        $config = $extension->getConfig(self::CONFIG_FILE);
+        $config['reviewLayout'] = 'default';
+        $config['displaySectionTitles'] = true;
+        $extension->setConfig(self::CONFIG_FILE, $config);
+    }
+
+    public function down(Schema $schema): void
+    {
+        // remove ReviewPanel.conf values
+        $extension = $this->getExtension();
+        $config = $extension->getConfig(self::CONFIG_FILE);
+        unset($config['reviewLayout']);
+        unset($config['displaySectionTitles']);
+        $extension->setConfig(self::CONFIG_FILE, $config);
+    }
+
+    /**
+     * @throws ExtensionException
+     *
+     * @return Extension
```

Comment:
```suggestion
```

Reply:
You already have the same specified on the level strick type hinting, any reason to repeat it twice?

Reply:
Ok, I see what you mean now :)

Reply:
You don't need to document the return type if you're type hinted it ": Extension" ;)